docs/machine-learning/README.md

```diff
@@ -19,7 +19,8 @@ The diagram below shows different phases in an ideal ML project. Due to practica
 
 ![Project flow](images/flow.png)
 
-* **[Envisioning](ml-problem-formulation-envisioning.md)**: Initial problem understanding, customer goals and objectives.
+* **[Envisioning](ml-problem-formulation-envisioning.md)**: Initial problem understanding, customer goals and objectives. See [template for envisioning outcomes summary document](ml-envisioning-summary-template.md).
```

Comment:
Note that all links in this list refer to points shown on the diagram. Therefore, I'm not sure the link to the template should come here. IMHO we could link it from the envisioning doc itself, or in the "general guidance" list below.